data/lua.js

```diff
@@ -399,6 +399,30 @@ If header is not found from the response, an empty string is returned.`
 		text: `Append prefix for log messages.`
 	},
 
+	{
+		name: 'set_always_log_source',
+		tags: [ 'event', 'event_passthrough' ],
+		text: `Add source path:line to the log messages.`
+	},
+
+	{
+		name: 'set_forced_debug',
+		tags: [ 'event' ],
+		added: {
+			'event_set_forced_debug_added': false
+		},
+		text: `Enable forced debugging for the event and its child events.`
```

Comment:
See #1153 : Remove added block and just add added info to markdown in text field:

`[[added,event_set_forced_debug_added]] Enable force debugging for the event and its child events.`